src/ape_plugins/_cli.py

```diff
@@ -168,7 +183,58 @@ def install(cli_ctx, skip_confirmation):
             # NOTE: Be *extremely careful* with this command, as it modifies the user's
             #       installed packages, to potentially catastrophic results
             # NOTE: This is not abstracted into another function *on purpose*
-            subprocess.call([sys.executable, "-m", "pip", "install", "--quiet", f"{package_name}"])
+            result = subprocess.call(
+                [sys.executable, "-m", "pip", "install", "--quiet", f"{package_name}"]
+            )
+            if result != 0:
+                cli_ctx.logger.error(f"Failed to add '{package_name}'.")
+                any_install_failed = True
+    if any_install_failed:
+        sys.exit(1)
+
+
+@cli.command(short_help="Uninstall all plugins in the local config file")
+@ape_cli_context()
+@skip_confirmation_option("Don't ask for confirmation to install the plugins")
+def uninstall(cli_ctx, skip_confirmation):
+    any_uninstall_failed = False
+    plugins = config.get_config("plugins") or []
+    for plugin in plugins:
+        module_name, package_name = extract_module_and_package_install_names(plugin)
+
+        available_plugin = module_name in github_client.available_plugins
+        installed_plugin = is_plugin_installed(module_name)
+
+        # if plugin is installed but not a 2nd class. It must be a third party
+        if installed_plugin and not available_plugin:
+            cli_ctx.logger.warning(
+                f"Plugin '{module_name}' is not installed but not in available plugins."
+                f" Please uninstall outside of Ape."
+            )
+            any_uninstall_failed = True
+            pass
+        # check for installed check the config.yaml
+        elif not installed_plugin:
+            cli_ctx.logger.warning(f"Plugin '{module_name}' is not installed.")
+            any_uninstall_failed = True
+            pass
+
+        # if plugin is installed and 2nd class. We should uninstall it
+        if installed_plugin and (available_plugin or skip_confirmation):
+            cli_ctx.logger.info(f"Uninstalling {package_name}...")
+            # NOTE: Be *extremely careful* with this command, as it modifies the user's
+            #       installed packages, to potentially catastrophic results
+            # NOTE: This is not abstracted into another function *on purpose*
+            result = subprocess.call(
+                [sys.executable, "-m", "pip", "uninstall", "--quiet", f"{package_name}"]
+            )
+            if result == 0:
+                cli_ctx.logger.success(f"Plugin '{package_name}' has been removed.")
```

Comment:
So since we can't tell in some situations that the packages was actually removed, causing this success message to be wrong, we can just check that if the plugin is installed and only echo success based on that.